src/evaluations/data/evaluation_configs.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Evaluation configurations."""
+import math
```

Comment:
possible to just use np types (e.g. `np.log(3)` as below to be consistent for both setup?)

Reply:
Thanks for noticing that! I convert all the np.log(3) to math.log(3) so as to save computation cost (though not that large).